.github/actions/docker-run/action.yml

```diff
@@ -9,6 +9,9 @@ inputs:
     description: 'Docker image architecture'
     required: false
     default: tt-metalium/ubuntu-20.04-amd64
+  docker_image_tag:
```

Comment:
@ttmchiou You will find this interesting :)

Reply:
We have similar logic in generate-docker-tag.
Can we either put this docker_image_tag into there or combine the logic so that it's all in one step with one arg?

Change seems to make this more wordy then it needs to be

Reply:
You can add an argument into generate-docker-tag to specify a tag i think as its set to either use 'latest' or branch name in an if statement. should be a matter of adding an extra if statement and feeding the argument in to docker-run